From 0a5e07b4a956501c5fd276662f73178a5eba69c0 Mon Sep 17 00:00:00 2001
From: Thomas D <whissi@whissi.de>
Date: Fri, 12 Dec 2014 16:12:01 +0100
Subject: [PATCH 2/2] Remove zlib status from summary

Because zlib is considered as required since commit 6c5b6a2ec,
we don't need to show if zlib support is enabled or not in the
settings summary.
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e09124c..5935a31 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1584,7 +1584,6 @@ echo
 echo "    Large file support enabled:               $enable_largefile"
 echo "    Networking support enabled:               $enable_inet"
 echo "    Regular expressions support enabled:      $enable_regexp"
-echo "    Zlib compression support enabled:         $enable_zlib"
 echo "    rsyslog runtime will be built:            $enable_rsyslogrt"
 echo "    rsyslogd will be built:                   $enable_rsyslogd"
 echo "    have to generate man pages:               $have_to_generate_man_pages"
-- 
2.2.0

